train:[Epoch:0]:                             [toatl_loss:51.59387]                             [regre_loss:47.07371]                             [cls_loss:4.52016]                             [R2:-1047.2997]                             [cls_acc:0.0116]: 100%|[31mâ–ˆ[39m| 2
 val :[Epoch:0]:                              [toatl_loss:21.94095]                              [regre_loss:17.36615]                              [cls_loss:4.57480]                              [R2:-794.6038]                              [cls_acc:0.0000]: 100%|
train:[Epoch:1]:                             [toatl_loss:13.85711]                             [regre_loss:9.31419]                             [cls_loss:4.54292]                             [R2:-521.0674]                             [cls_acc:0.0058]: 100%|[31mâ–ˆ[39m| 2/2
 val :[Epoch:1]:                              [toatl_loss:5.85864]                              [regre_loss:1.28388]                              [cls_loss:4.57476]                              [R2:-32.0396]                              [cls_acc:0.0000]: 100%|[34mâ–ˆ[39m|
train:[Epoch:2]:                             [toatl_loss:5.82848]                             [regre_loss:1.27997]                             [cls_loss:4.54851]                             [R2:-28.8455]                             [cls_acc:0.0116]: 100%|[31mâ–ˆ[39m| 2/2 [
 val :[Epoch:2]:                              [toatl_loss:5.46779]                              [regre_loss:0.89306]                              [cls_loss:4.57473]                              [R2:-36.5836]                              [cls_acc:0.0000]: 100%|[34mâ–ˆ[39m|
train:[Epoch:3]:                             [toatl_loss:5.39699]                             [regre_loss:0.85344]                             [cls_loss:4.54354]                             [R2:-34.3988]                             [cls_acc:0.0116]: 100%|[31mâ–ˆ[39m| 2/2 [
 val :[Epoch:3]:                              [toatl_loss:5.26857]                              [regre_loss:0.69385]                              [cls_loss:4.57472]                              [R2:-35.1797]                              [cls_acc:0.0000]: 100%|[34mâ–ˆ[39m|
train:[Epoch:4]:                             [toatl_loss:5.26489]                             [regre_loss:0.72227]                             [cls_loss:4.54262]                             [R2:-28.9035]                             [cls_acc:0.0116]: 100%|[31mâ–ˆ[39m| 2/2 [
 val :[Epoch:4]:                              [toatl_loss:4.65197]                              [regre_loss:0.07725]                              [cls_loss:4.57472]                              [R2:-2.2275]                              [cls_acc:0.0000]: 100%|[34mâ–ˆ[39m| 1
train:[Epoch:5]:                             [toatl_loss:5.17614]                             [regre_loss:0.61181]                             [cls_loss:4.56432]                             [R2:-15.4478]                             [cls_acc:0.0233]: 100%|[31mâ–ˆ[39m| 2/2 [
 val :[Epoch:5]:                              [toatl_loss:6.04071]                              [regre_loss:1.46600]                              [cls_loss:4.57472]                              [R2:-83.0423]                              [cls_acc:0.0000]: 100%|[34mâ–ˆ[39m|
train:[Epoch:6]:                             [toatl_loss:5.93709]                             [regre_loss:1.39356]                             [cls_loss:4.54352]                             [R2:-66.1180]                             [cls_acc:0.0116]: 100%|[31mâ–ˆ[39m| 2/2 [
 val :[Epoch:6]:                              [toatl_loss:4.86801]                              [regre_loss:0.29329]                              [cls_loss:4.57472]                              [R2:-8.1862]                              [cls_acc:0.0000]: 100%|[34mâ–ˆ[39m| 1
train:[Epoch:7]:                             [toatl_loss:4.94117]                             [regre_loss:0.37665]                             [cls_loss:4.56452]                             [R2:-12.7106]                             [cls_acc:0.0000]: 100%|[31mâ–ˆ[39m| 2/2 [
 val :[Epoch:7]:                              [toatl_loss:5.19806]                              [regre_loss:0.62334]                              [cls_loss:4.57472]                              [R2:-19.9963]                              [cls_acc:0.0000]: 100%|[34mâ–ˆ[39m|
train:[Epoch:8]:                             [toatl_loss:5.20777]                             [regre_loss:0.65921]                             [cls_loss:4.54856]                             [R2:-18.4001]                             [cls_acc:0.0116]: 100%|[31mâ–ˆ[39m| 2/2 [
 val :[Epoch:8]:                              [toatl_loss:4.81512]                              [regre_loss:0.24041]                              [cls_loss:4.57472]                              [R2:-7.6906]                              [cls_acc:0.0000]: 100%|[34mâ–ˆ[39m| 1
train:[Epoch:9]:                             [toatl_loss:4.85546]                             [regre_loss:0.29107]                             [cls_loss:4.56439]                             [R2:-8.5139]                             [cls_acc:0.0174]: 100%|[31mâ–ˆ[39m| 2/2 [0
 val :[Epoch:9]:                              [toatl_loss:4.79404]                              [regre_loss:0.21932]                              [cls_loss:4.57472]                              [R2:-6.3407]                              [cls_acc:0.0000]: 100%|[34mâ–ˆ[39m| 1
train:[Epoch:10]:                             [toatl_loss:4.91610]                             [regre_loss:0.37372]                             [cls_loss:4.54239]                             [R2:-11.9867]                             [cls_acc:0.0116]: 100%|[31mâ–ˆ[39m| 2/2
  0%|[34m                                                                                                                                                                                                                                            [39m| 0/1 [00:00<?, ?it/s]Exception ignored in: <function _releaseLock at 0x76c154ee6d30>
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/envs/ndt2/lib/python3.9/logging/__init__.py", line 227, in _releaseLock
    def _releaseLock():
KeyboardInterrupt:
  0%|[34m                                                                                                                                                                                                                                            [39m| 0/1 [00:05<?, ?it/s]
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/envs/ndt2/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1120, in _try_get_data
    data = self._data_queue.get(timeout=timeout)
  File "/home/ubuntu/anaconda3/envs/ndt2/lib/python3.9/multiprocessing/queues.py", line 114, in get
    raise Empty
_queue.Empty
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/mnt/c/gaochao/CODE/BCI/ndt47/train4traj.py", line 362, in <module>
    main(args)
  File "/mnt/c/gaochao/CODE/BCI/ndt47/train4traj.py", line 334, in main
    evaluate(model, criterion, val_loader, device, epoch, display_config={'color':'blue', 'phase':'val'})
  File "/home/ubuntu/anaconda3/envs/ndt2/lib/python3.9/site-packages/torch/autograd/grad_mode.py", line 27, in decorate_context
    return func(*args, **kwargs)
  File "/mnt/c/gaochao/CODE/BCI/ndt47/train4traj.py", line 130, in evaluate
    for idx, samples in enumerate(pbar):
  File "/home/ubuntu/anaconda3/envs/ndt2/lib/python3.9/site-packages/tqdm/std.py", line 1181, in __iter__
    for obj in iterable:
  File "/home/ubuntu/anaconda3/envs/ndt2/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 628, in __next__
    data = self._next_data()
  File "/home/ubuntu/anaconda3/envs/ndt2/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1316, in _next_data
    idx, data = self._get_data()
  File "/home/ubuntu/anaconda3/envs/ndt2/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1282, in _get_data
    success, data = self._try_get_data()
  File "/home/ubuntu/anaconda3/envs/ndt2/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1133, in _try_get_data
    raise RuntimeError('DataLoader worker (pid(s) {}) exited unexpectedly'.format(pids_str)) from e
RuntimeError: DataLoader worker (pid(s) 3641071, 3641103, 3641135, 3641167, 3641199, 3641231) exited unexpectedly